src/cljs/main/broadfcui/page/method_repo_NEW/method_repo_table.cljs

```diff
@@ -63,41 +64,48 @@
-                                   {:className (when allow-modals? "underline-on-hover")
-                                    :style {:fontSize "80%" :color "black"}
-                                    :onClick (when allow-modals?
-                                               (fn [e]
-                                                 (.preventDefault e)
-                                                 (swap! state assoc :editing-namespace namespace)))}
-                                   namespace]
-                                  [:div {:style {:fontWeight 600}} name]))}
-                              {:header "Synopsis" :initial-width 700
+                                 (let [method-id (utils/restructure namespace name)]
+                                   (links/create-internal
+                                     (utils/deep-merge
+                                      {:data-test-id (str "method-link-" namespace "-" name)
+                                       :style {:display "block" :marginTop -4}}
+                                      (if workspace-id
+                                        {:onClick #(nav-method
+                                                    {:label (str namespace "/" name)
+                                                     :component MethodDetails
+                                                     :props (utils/restructure method-id nav-method workspace-id)})}
+                                        {:href (nav/get-link :method-loader method-id)}))
+                                     [:span
+                                      {:className (when-not workspace-id "underline-on-hover")
+                                       :style {:fontSize "80%" :color "black"}
+                                       :onClick (when-not workspace-id
+                                                  (fn [e]
+                                                    (.preventDefault e)
+                                                    (swap! state assoc :editing-namespace namespace)))}
+                                      namespace]
+                                     [:div {:style {:fontWeight 600}} name])))}
+                              {:header "Synopsis" :initial-width 475
                                :column-data :synopsis
                                :sort-by string/lower-case}
-                              {:header "Owners" :initial-width 175
+                              {:header "Owners" :initial-width 225
                                :column-data :managers
                                :as-text (partial string/join ", ")}
                               {:header "Snapshots" :initial-width 94 :filterable? false
                                :column-data :numSnapshots}
-                              {:header "Configurations" :initial-width 117 :filterable? false
+                              {:header "Configurations" :initial-width :auto :filterable? false
```

Comment:
`:auto` behaves poorly when the table is too wide for the view, which easily happens here.

Reply:
I can't get it to misbehave in this instance...